src/flexbox/nasa.css

```diff
@@ -0,0 +1,27 @@
+body{
+    width: 850px;
+    margin: 0 auto;
+}
+#back{
+    background-image: url("../../photos/nasa_image.jpg");
+    width: 100%;
+    height: 100%;
```

Comment:
add `background-size: contain;` so the image only gets as big as the container it's in.
also i'm gonna have you move `<h1> NASA </h1>` into your `<header>` div. when you do add flex so you can center the image!
`display: flex;`
`justify-content: center;`
 `align-items: center;`